### PR TITLE
Fix login and user profile design 

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@bruskiza ready for review :)

login-smart
login-feature

File: tuneme/templates/base.html [hunk 1]
```diff
@@ -53,9 +56,10 @@
           <div class="title">
             <h3>{% trans "Sign out" %}</h3>
           </div>
-          <p>{% trans "Lorem ipsum dolor sit amet, consectetur adipiscing elit." %}</p>
+
           <div class="block turq">
           <div class="signin-form">
+            <p>{% trans "Lorem ipsum dolor sit amet, consectetur adipiscing elit." %}</p>
```
Comment (bruskiza) [inline tuneme/templates/base.html]: Lorem ipsum to be replaced with: "Thanks for spending time with us. See you soon!"

Comment (bruskiza) [inline tuneme/templates/base.html]: Lorem ipsum to be replaced with: "Welcome to TuneMe! Not a member yet? Join us by signing up below. Already signed-up? Then sign in."

**Comment by bruskiza** [comment]
:+1: when lorem ipsum removed :)

**Pull request merged**